src/index.html

```diff
@@ -493,5 +493,23 @@ <h2>What's that data type?</h2>
       </div>
     </div>
 
+    <div class="exercise">
+      <div class="title">
+        <h2>Python in DataCamp Light 3</h2>
+        <p>In the HTML code for this exercise you can specify the packages to import and the Python version you want to use.</p>
+      </div>
+      <div data-datacamp-exercise data-lang="python" lang-version="3.6" packages="pandas==0.24.1, alabaster==0.7.12" data-height="auto">
```

Comment:
I suggest prepending all attributes with `data-`

Reply:
Yes good idea, done in the last commit